### PR TITLE
fix: update_path_script return normalized path for Linux platform

### PR DESCRIPTION
**Pull request by ChugunovRoman**

### CODE REVIEW

File: src/Common/PlatformLinux.inl [hunk 1]
```diff
@@ -1097,6 +1097,15 @@ inline void restore_path_separators(char * path)
...
 }
 
+/** Replace windows path separator on Linux path separator 
+ * @brief normalize_path_separators
+ * @param path
+ */
+inline void normalize_path_separators(char * path)
+{
+    while (char* sep = strchr(path, '\\')) *sep = '/'; //
+}
```
Comment (Xottab-DUTY) [inline src/Common/PlatformLinux.inl]: What? This is a full duplicate of `convert_path_separators` function!

File: src/xrGame/fs_registrator_script.cpp [hunk 2]
```diff
@@ -11,6 +11,9 @@ LPCSTR update_path_script(CLocatorAPI* fs, LPCSTR initial, LPCSTR src)
     string_path temp;
     shared_str temp_2;
     fs->update_path(temp, initial, src);
+#ifdef XR_PLATFORM_LINUX
```
Comment (Xottab-DUTY) [inline src/xrGame/fs_registrator_script.cpp]:
```suggestion
#ifndef XR_PLATFORM_WINDOWS
```

Comment (ChugunovRoman) [reply src/xrGame/fs_registrator_script.cpp]: Fixed

File: src/Common/PlatformLinux.inl (hunk 1 above)
Comment (ChugunovRoman) [reply src/Common/PlatformLinux.inl]: I didn't see this function :(
Replaced

**Review by Xottab-DUTY — APPROVED** [review]

File: src/xrGame/fs_registrator_script.cpp [hunk 3]
```diff
@@ -11,6 +11,9 @@ LPCSTR update_path_script(CLocatorAPI* fs, LPCSTR initial, LPCSTR src)
     string_path temp;
     shared_str temp_2;
     fs->update_path(temp, initial, src);
+#ifndef XR_PLATFORM_WINDOWS
```
Comment (Xottab-DUTY) [inline src/xrGame/fs_registrator_script.cpp]:
```suggestion
#ifndef XR_PLATFORM_WINDOWS // XXX: replace with runtime case-sensitivity flag check
```

**Comment by Xottab-DUTY** [comment]
@ChugunovRoman, this change works ok with vanilla COP and CS?

**Comment by ChugunovRoman** [comment]
> @ChugunovRoman, this change works ok with vanilla COP and CS?

Yes, of course. This fixes using of system paths on Linux for scripts. Before this change scripts get path with '\\' on linux, after - with '/'
For windows this change nothing

**Pull request merged**